Automate/Tests/Speckle.Automate.Sdk.Tests.Integration/Speckle.Automate.Sdk.Tests.Integration.csproj

```diff
@@ -10,6 +10,10 @@
         <NoWarn>$(NoWarn);CA2007</NoWarn>
```

Comment:
One last quick change (Sorry this one is on me, I forgot to remove this earlier).
this line is no longer needed, and can be removed (since CA2007 is already supressed in our Build.targets for test projects)

Reply:
done and no any following warning because of this